### PR TITLE
4382: pagination for case inventory report

### PR DESCRIPTION
**Pull request by rachaelparris**

### CODE REVIEW

File: shared/src/business/entities/cases/Case.test.js [hunk 1]
```diff
@@ -1335,6 +1335,34 @@ describe('Case entity', () => {
...
+      const myCase = new Case(
+        {
+          ...MOCK_CASE,
+          docketNumber: '105-19',
+        },
+        {
+          applicationContext,
+        },
+      );
+      expect(myCase.generateSortableDocketNumber()).toEqual(19105);
```
Comment (kkoskelin) [inline shared/src/business/entities/cases/Case.test.js]: Will these sortable docket numbers do what we want? e.g. compare 225-19 vs 1004-19? Will we need to zero-pad?

Comment (rachaelparris) [reply shared/src/business/entities/cases/Case.test.js]: Oh good point! We probably should zero-pad.

File: shared/src/business/useCaseHelper/caseInventoryReport/getCaseInventoryReport.js [hunk 2]
```diff
@@ -13,6 +13,8 @@ const { get } = require('lodash');
 exports.getCaseInventoryReport = async ({
   applicationContext,
   associatedJudge,
+  from = 0,
+  pageSize,
```
Comment (kkoskelin) [inline shared/src/business/useCaseHelper/caseInventoryReport/getCaseInventoryReport.js:19]: jsdoc updates needed.

File: shared/src/business/useCaseHelper/caseInventoryReport/getCaseInventoryReport.test.js [hunk 3]
```diff
@@ -150,6 +154,73 @@ describe('getCaseInventoryReport', () => {
     });
   });
 
+  it('calls the search function with a default page size if one is not provided', async () => {
+    searchSpy = jest.fn(async () => {});
```
Comment (kkoskelin) [inline shared/src/business/useCaseHelper/caseInventoryReport/getCaseInventoryReport.test.js]: fyi you can `jest.fn().mockResolvedValue()`

File: shared/src/business/useCases/caseInventoryReport/getCaseInventoryReportInteractor.js [hunk 4]
```diff
@@ -16,6 +16,8 @@ const { UnauthorizedError } = require('../../../errors/errors');
 exports.getCaseInventoryReportInteractor = async ({
   applicationContext,
   associatedJudge,
+  from,
+  pageSize,
```
Comment (kkoskelin) [inline shared/src/business/useCases/caseInventoryReport/getCaseInventoryReportInteractor.js:22]: jsdoc

File: shared/src/proxies/reports/getCaseInventoryReportProxy.js [hunk 5]
```diff
@@ -13,9 +13,17 @@ const { get } = require('../requests');
 exports.getCaseInventoryReportInteractor = ({
   applicationContext,
   associatedJudge,
+  page = 1,
```
Comment (kkoskelin) [inline shared/src/proxies/reports/getCaseInventoryReportProxy.js:17]: jsdoc

**Review by matthopson — APPROVED** [review]

File: shared/src/business/entities/cases/Case.js [hunk 6]
```diff
@@ -688,6 +691,10 @@ joiValidationDecorator(
       .optional()
       .allow(null)
       .description('When the case was sealed from the public.'),
+    sortableDocketNumber: joi
+      .number()
+      .required()
+      .description('A sortable representation of the docket number'),
```
Comment (ericsorenson) [inline shared/src/business/entities/cases/Case.js]: Can we please add to the description an explanation that this value is autogenerated by the constructor? This field could be confusing for the migration.

**Review by kkoskelin — APPROVED** [review]
update some jsdocs and then 👍

**Review by ericsorenson — APPROVED** [review]

**Pull request merged**